File: Python developer/step03/3.01/3.1.02.py
'''

Представим, что вы разработчик серверных приложений. У вас есть потребность в измерении количества времени, которое проводят ваши пользователи на сайте. 
Фронтенд посылает вам время в минутах, которое пользователь провел на сайте. Вам в ответ нужно узнать сколько в этих минутах часов и минут.

Или иными словами: 

Ваша программа принимает одну переменную m – количество минут, которое пользователь провел на сайте. Вам нужно определить сколько в этих минутах часов и оставшихся минут.

Sample Input 1:

153

Sample Output 1:

2 33

Sample Input 2:

90

Sample Output 2:

1 30

Sample Input 3:

60

Sample Output 3:

1 0

Sample Input 4:

0

Sample Output 4:

0 0

Sample Input 5:

156

Sample Output 5:

2 36

'''

m = int(input())
print(m//60,m%60)

File: Fundamentals of C programming/step03/3.1/3.1.02.c
/*

Дополните спецификаторы формата соответствующими модификаторами, чтобы вывод на экран выглядел следующим образом:
 

||-----|-----|-----|-----|| || act | one | two | res || ||=====+=====+=====+=====|| || +|3 |4 |00007|| || -| 3| 4|-0001|| || *| 3|4 |00012|| ||/ |3 | 4|0.750|| ===========================

Sample Input:


Sample Output:

||-----|-----|-----|-----||
|| act | one | two | res ||
||=====+=====+=====+=====||
||    +|3    |4    |00007||
||    -|    3|    4|-0001||
||    *|    3|4    |00012||
||/    |3    |    4|0.750||
===========================


*/

#include <stdio.h> 
int main(void){
  int a=3, b=4;
  double res = 0.75;

  printf("||-----|-----|-----|-----||\n");
  printf("|| act | one | two | res ||\n");
  printf("||=====+=====+=====+=====||\n");
  printf("||%5c|%-5d|%-5d|%5.5d||\n",'+',a,b,a+b);
  printf("||%5c|%5d|%5d|%5.4d||\n",'-',a,b,a-b);
  printf("||%5c|%5d|%-5d|%5.5d||\n",'*',a,b,a*b);
  printf("||%-5c|%-5d|%5d|%1.3f||\n",'/',a,b,res);
  printf("===========================");

  return(0);
}
